core/src/main/scala/org/apache/spark/scheduler/TaskInfo.scala

```diff
@@ -20,6 +20,8 @@ package org.apache.spark.scheduler
 import scala.collection.mutable.ListBuffer
 
 import org.apache.spark.annotation.DeveloperApi
+import org.apache.spark.TaskState
```

Comment:
alphabetization (capital letters should come before lowercase -- so this and the import below it should both come above org.apache.spark.annotation.DeveloperApi)